community/urls.py

```diff
@@ -127,6 +128,12 @@ def get_organization():
         distill_func=get_index,
         distill_file='contributors/index.html',
     ),
+    distill_url(
+        r'meta-review/$', meta_review_index,
```

Comment:
IMO, keep it `meta_review`, as your app name is also the same.

Reply:
snake_case is a bit unusual for url.

Reply:
As what we discussed here https://gitter.im/coala/community?at=5b0d5fe8b6eece791dc0de4e, snake case is a bit unusual for URL 😝 
Any voice is welcome :P